onnxruntime/core/providers/webnn/builders/impl/batch_norm_op_builder.cc

```diff
@@ -34,17 +34,18 @@ Status BatchNormalizationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_bu
                                                           const Node& node,
                                                           const logging::Logger& /* logger */) const {
   const auto& input_defs = node.InputDefs();
-  ::wnn::Operand input = model_builder.GetOperand(input_defs[0]->Name());
-  ::wnn::BatchNormOptions options;
-  options.scale = model_builder.GetOperand(input_defs[1]->Name());
-  options.bias = model_builder.GetOperand(input_defs[2]->Name());
-  ::wnn::Operand mean = model_builder.GetOperand(input_defs[3]->Name());
-  ::wnn::Operand variance = model_builder.GetOperand(input_defs[4]->Name());
+  emscripten::val input = model_builder.GetOperand(input_defs[0]->Name());
+  emscripten::val options = model_builder.GetBuilder().call<emscripten::val>("MLBatchNormalizationOptions");
```

Comment:
WebNN doesn't have a method named `MLBatchNormalizationOptions`, just new a empty val object for options.